src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import logoImage from "../../images/XriiLogo/gradient logo.svg";
+import { Link } from "react-router-dom";
+import paths from "../../Paths/paths";
+import "./style.scss";
+
+const Header = () => {
+  return (
+    <header className="header_container">
+      <div className="header_logo">
+        <Link to={paths.HOME}>
+          <img src={logoImage} alt="companyLogo" />
+        </Link>
+      </div>
+      <nav className="header_navigation">
+        <Link to={paths.CASE}>Case studies</Link>
+        <Link to={paths.PRICE}>Pricing</Link>
+          <Link to={paths.ABOUT}>About</Link>
+        <Link to={paths.BLOG}>Blog</Link>
+        <Link to={paths.HOME}></Link>
```

Comment:
I wonder if this last one is needed, as the first one -logo is Home. And this one does not even have any content/ children

Reply:
Changed to <Link to={paths.HOME}>Home</Link>. Added  link "HOME" to navigation.